tests/test_handlers.py

```diff
@@ -205,7 +200,7 @@ async def test_get_object(self):
         response = await self.client.get(url)
         assert response.status == 200
```

Comment:
a bit weird to use `assert` in one place and `self.assertEquals` in the next one :confused:

Reply:
Yeah that I agree, which way to go? Assert or assertEquals?

Reply:
`assertEquals` as we are using the unit test class